### ISSUE TITLE
Question about way to create sentences

### ISSUE THREAD

**Issue by shtu-ryan** [GuangyuWangLab2021/Loki#29]
In the paper, it is only said that top 50 expressed genes are selected and combined into a sentence. What is the exact meaning of the top expressed gene? Is it to simply use expression value after normalization or there is other way to do this?

And I have another question: Did you try to do contrast with multiple spots and a larger image? If so, how is the sentence created?

Thank you for the great work!

**Comment by osakemon** [comment]
Hi shtu-ryan,

Thanks for the questions!

1. We essentially ranked genes by their normalized expression and take the top 50 per spot to form the sentence. Please see the Methods - Training dataset curation section of the paper for more details.
2. In the current version we only contrast a single spot with its paired image patch.

Let us know if you have further questions.

Best,
Weiqing
